centreon/src/Core/Platform/Application/UseCase/UpdateVersions/UpdateVersionsException.php

```diff
@@ -87,6 +87,14 @@ public static function errorWhenApplyingUpdate(
      */
     public static function errorWhenApplyingPostUpdate(\Throwable $ex): self
     {
-        return new self(_('An error occurred when applying post update actions'), 0, $ex);
+        return new self(
+            sprintf(
+                '%s: %s',
+                _('An error occurred when applying post update actions'),
+                $ex->getMessage()
```

Comment:
It's dangerous to expose error messages whose source is unknown.
That's what logs are for, among other things.